test/floki_test.exs

```diff
@@ -1576,14 +1583,19 @@ defmodule FlokiTest do
 
     [elements | _] = Floki.find(html, "body > div")
 
-    assert [
-             {"span", [], ["child 1"]},
-             {"span", [], ["child 2"]}
-           ] = Floki.children(elements, include_text: false)
+    expected = [
+      {"span", [], ["child 1"]},
+      {"span", [], ["child 2"]}
+    ]
+
+    assert Floki.children(elements, include_text: false) == expected
+    assert Floki.children(elements, include_text: false, unknown_option: true) == expected
```

Comment:
Same question here. I think we could filter out unknown options - maybe with `Keyword.validate!/2`, but we would need to remove support for Elixir 1.12. WDYT?

Reply:
This is to have coverage in the fallback when we are not able to pattern match the include_text value directly
```
  def children({_, _, _} = html_node, opts) do
    children(html_node, include_text: opts[:include_text])
  end
```
Not sure if there's a better way to cause this to fail besides `Keyword.validate!/2`, I didn't even knew about this function

Reply:
OK, I think I will try to refactor that once I drop support for Elixir 1.12. Thanks!